api-specificatie/openapi.yaml

```diff
@@ -0,0 +1,291 @@
+        '415':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/415"
+        '429':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/429"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
+      tags:
+      - volgindicatiepersonen
+  /volgindicatiepersonen/{burgerservicenummer}:
+    delete:
+      operationId: DeleteVolgindicatiePersonen
+      description: |
+        Beëindigen volgindicatie op burgerservicenummer voor een abonnee.
+      parameters:
+        - in: path
+          name: burgerservicenummer
+          required: true
+          description: "Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet\
+            \ algemene bepalingen burgerservicenummer. Alle nummers waarvoor geldt\
+            \ dat, indien aangeduid als (s0 s1 s2 s3 s4 s5 s6 s7 s8), het resultaat\
+            \ van (9*s0) + (8*s1) + (7*s2) +...+ (2*s7) - (1*s8) deelbaar is door\
+            \ elf. Er moeten dus 9 cijfers aanwezig zijn."
+          explode: false
+          schema:
+            $ref: "#/components/schemas/Burgerservicenummer"
+      responses:
+        200:
+          description: "OK"
+          headers:
+            api-version:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/api_version"
+            warning:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/warning"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VolgindicatiePersonen"
+        '204':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/204"
+        '404':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/404"
+      tags:
+      - volgindicatiepersonen
+  /gewijzigdepersonen:
+    get:
+      operationId: GetGewijzigdePersonen
+      description: |
+        Opvragen lijst met gewijzigdepersonen op relevante burgerservicenummers.
+      parameters:
+        - in: query
+          name: vanaf
+          required: false
+          explode: false
+          schema:
+            type: "string"
+            format: "date"
```

Comment:
date of date-time?

Reply:
Kunnen we daar nu al iets zinnigs over zeggen?
In dat geval kan ik het evt. aanpassen anders zou ik zeggen dat dit voor nu een commit niet in de weg hoeft te staan. We kunnen het op een later moment nog wijzigen.